packages/ui-components/src/components/Readme/utils.ts

```diff
@@ -21,9 +21,6 @@ marked.setOptions({
 });
 
 export function parseReadme(readme: string): string | void {
-  if (typeof readme === 'string') {
```

Comment:
It might be the case this is undefined, but not 100% sure.

Reply:
The parameter is `readme: string` so if it's indeed undefined, it would fail somewhere else (where TS isn't strict).